README.md

```diff
@@ -18,6 +18,8 @@
 
 Suomenkieliset käyttöohjeet löydät [täältä](./README-Finnish.md)
 
+Türkçe Talimatları [burada](./README-Turkish.md) bulabilirsiniz.
```

Comment:
Not that I know any Turkish, but I ran this through Google Translate and saw "English Instructions can be found here" - maybe this should be saying that the Turkish translations are here (rather than English)?

Let me know if I'm wrong, this is just based on my checks in Google Translate.

Similar situation for the Finnish.md file.

Reply:
Sweet, just wanted to check 👌